### PR TITLE
Sifchain: adding Rowan asset to the osmosis-1 network asset list.

### PR DESCRIPTION
**Pull request by gzukel**
Added Rowan.

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -442,6 +442,34 @@
...
+        },
+        {
+          "denom": "rowan",
+          "exponent": 10,
+          "aliases": []
+        }
+      ],
+      "base": "ibc/8B5FDA7B727F8E0C4B6E73A1DF529CE4C63208FD66FC207238A9ECB872C70419",
+      "display": "rowan",
+      "symbol": "ROWAN",
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:466]:
```suggestion
      "name": "Sifchain",
      "symbol": "ROWAN",
```

**Comment by gzukel** [comment]
@IbrarMakaveli Thanks for the changes. Applied. Appreciate it.

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -443,6 +443,35 @@
...
+      "base": "ibc/8B5FDA7B727F8E0C4B6E73A1DF529CE4C63208FD66FC207238A9ECB872C70419",
+      "display": "rowan",
+      "name": "Sifchain",
+      "symbol": "ROWAN",
+      "ibc": {
+        "source_channel": "channel-17",
+        "dst_channel": "channel-47"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/rowan.png"
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:472]:
```suggestion
        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/rowan.png",
```

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]

**Comment by sunnya97** [comment]
Moving to #33  to fix merge conflicts

File: osmosis-1/osmosis-1.assetlist.json [hunk 3]
```diff
@@ -443,6 +443,35 @@
...
+        {
+          "denom": "ibc/8B5FDA7B727F8E0C4B6E73A1DF529CE4C63208FD66FC207238A9ECB872C70419",
+          "exponent": 0,
+          "aliases": [
+            "xrowan"
+          ]
+        },
+        {
+          "denom": "rowan",
+          "exponent": 10,
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:459]: Guys, I'm pretty sure this is wrong.  Doesn't rowan use 18 decimal points of precission.

**Pull request closed without merging**